### DISCUSSION TITLE
Unable to connect my hub.

### DISCUSSION THREAD

**Discussion by acbeverage** [pybricks/support#1508] [Q&A]
I have the Technic Hub and I am able to connect it to my laptop (Windows10 HP) with Intel BT by opening the BT and add new devices. It shows up as Technic Hub and connects. Yet when I go into https://code.pybricks.com/ and try and connect it does not show up. I removed it out of my devices in Bluetooth and the same result. I am using Microsoft edge. I am able to connect with my phone and the Technic app so I do not think this is a battery or hub issue. Any suggestions or advice?

**Comment by laurensvalk** [comment]
Did you already install Pybricks on the hub? 

Before you can use the hub, you have to install the Pybricks firmware. Click on the gear icon to see the settings menu, and then click 'install Pybricks firmware' and follow the steps.

I think it also suggests doing this when trying it via the Bluetooth button did not work. Let me know if you're not getting these messages.

**Reply by acbeverage** [reply]
No I cannot get Pybricks to recognize the hub it just spins on the app and the hub just blinks it never goes solid.

**Reply by laurensvalk** [reply]
Which color does the light blink?

**Reply by acbeverage** [reply]
I have had it on white and flashing or held down with the light purple flashing it doesn't add it in Pybricks

**Reply by laurensvalk** [reply]
Blinking white means that the Pybricks firmware was not installed yet.

Have you followed the firmware installation steps? Can you share a screenshot?

What I mean is that you have to click on 

![image](https://github.com/pybricks/support/assets/12326241/a4ded7d2-de35-4964-ba2e-311498b8b040)

(Don't click on the Bluetooth button)
